src/DataTable/propTypes.js

```diff
@@ -125,6 +125,7 @@ export const propTypes = {
   paginationTotalRows: PropTypes.number,
   paginationPerPage: PropTypes.number,
   paginationRowsPerPageOptions: PropTypes.array,
+  paginationPosition: PropTypes.oneOf(['bottom', 'top']),
```

Comment:
We should consider in a future version `both` so pagination renders on top and bottom.  I am not sure how this would look but this PR is a good setup for that later 😉